source/mir/math/sum.d

```diff
@@ -472,14 +494,17 @@ struct Summator(T, Summation summation)
 
     @attr:
 
-    static if (summation == Summation.pairwise)
+    static if (summation == Summation.pairwise) {
+        import std.complex: Complex;
```

Comment:
the last one std.complex import

Reply:
@jmh530 please remove this one import too

Reply:
@9il Ah, sorry, fixed.